### PR TITLE
Radiation from contaminated objects no longer produces power in rad collectors

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

Goodbye rad-duping with 500000000000 matter bins being a meta strat for making quadrillions of power!! No more "but its effective!" lagging the server!!

This might result in a decrease to effective power output of radiation-based engines but I doubt it will be much

# Wiki Documentation

Radiation collectors ONLY produce power from radiation sources, not contaminated objects

# Changelog
:cl:  
rscdel: Radiation collectors no longer produce power from contamination
/:cl:

### CODE REVIEW

**Comment by ThatLing** [comment]
Did you check engine power output before and after?

File: code/game/objects/items/devices/geiger_counter.dm [hunk 1]
```diff
@@ -111,7 +111,7 @@
 	loop.last_radiation = radiation_count
 	loop.start()
 
-/obj/item/geiger_counter/rad_act(amount)
+/obj/item/geiger_counter/rad_act(amount, collectable_radiation)
```
Comment (ThatLing) [inline code/game/objects/items/devices/geiger_counter.dm:114]: Make geiger counters show the amount of collectable vs non-collectable radiation

**Comment by SomeguyManperson** [comment]
roundstart supermatter with basic nitrogen setup no contamination power
image

roundstart supermatter with basic nitrogen setup with contamination power
image

generic supermatter doesn't seem to get enough radiation to produce contamination, at least with the shutters open


roundstart singulo with basic engine setup no contamination power
image

roundstart singulo with basic engine setup with contamination power
image

halves the power output of the singularity, it's still in the MWs so it doesn't particularly matter

**Comment by Ethan4303** [comment]
Personally I'm biased, but I hate this pr so much frfr no cap, let us have our manifested atmosia God in turbine >:(

**Comment by ThatLing** [comment]
Should buff singulo to compensate

**Comment by MajManatee** [comment]
uh oh, cringe.

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**